ui/src/alerting/components/CheckHistory.tsx

```diff
@@ -0,0 +1,113 @@
+// Libraries
+import React, {useMemo, FC} from 'react'
+import {Page} from '@influxdata/clockface'
+import {connect} from 'react-redux'
+
+// Components
+import {ResourceIDsContext} from 'src/alerting/components/AlertHistoryIndex'
```

Comment:
this isn't a component, lets put it in another section plz

Reply:
Found it imported in other files as context, so added that section